src/containers/offer-page/chat-dialog-window/ChatDialogWindow.tsx

```diff
@@ -89,14 +101,40 @@ const ChatDialogWindow: FC<ChatDialogWindow> = ({ chatInfo }) => {
     messagesListWithDate
   )
 
-  const questionsList = questions.map((el) => (
-    <Typography key={el.question} sx={styles.question}>
-      {t(el.question)}
-    </Typography>
-  ))
+  const questionsList = creatingChat ? (
+    <Box sx={styles.chatCreateBox}>
+      <Loader size={20} />
+      <Typography sx={styles.question}>{t('chatPage.creating')}</Typography>
+    </Box>
+  ) : (
+    questions.map((el) => (
+      <Typography key={el.question} sx={styles.question}>
+        {t(el.question)}
+      </Typography>
+    ))
+  )
 
   const closeChatWindow = () => setChatInfo(null)
 
+  const openChatInNewTab = (chatId: unknown) => {
+    localStorage.setItem('currentChatId', chatId as string)
```

Comment:
I guess you don't need to set id in localStorage. You can try to take it from context.